tests/integration/test_quorum_inserts/test.py

```diff
@@ -311,12 +302,12 @@ def test_insert_quorum_with_ttl(started_cluster):
     zero.query(create_query)
     first.query(create_query)
 
-    print("Stop fetches for test_insert_quorum_with_ttl at first replica.")
-    first.query("SYSTEM STOP FETCHES test_insert_quorum_with_ttl")
+    print(f"Stop fetches for {table_name} at first replica.")
```

Comment:
I prefer `logging.debug` in integration tests, so these line are also printed to pylog which we archive on CI runs. However as these changes didn't make it worse, I don't mind if it stay as is, but if you have to change something in the PR again, it might worth a quck search and replace.

Reply:
Let's merge this one asap because broken master is irritating and after that we can make another PR